Test/cli/errorLimit.dfy

```diff
@@ -1,4 +1,4 @@
-// RUN: %exits-with 4 %baredafny verify %args --error-limit=2 "%s" > "%t"
+// RUN: %exits-with 4 %baredafny verify %args --error-limit=0 "%s" > "%t"
```

Comment:
This change might make sense to include for 3.x, as well. Any error limit other than 0 can lead to non-determinism.